client/src/controllers/ZoneController.test.js

```diff
@@ -0,0 +1,137 @@
+import { Application } from '@hotwired/stimulus';
+import { ZoneController } from './ZoneController';
+
+describe('ZoneController', () => {
+  let app;
```

Comment:
```suggestion
  let application;
```

Just a nitpick, can we use `application`, I know we are inconsistent in the tests but we are trying to align towards `application` not `app`.